content/_index.md

```
@@ -5,6 +5,15 @@ title = ""
 
 # Latest Posts
 
+<a href="/development/ai-wont-kill-your-job-your-customers-using-ai-will"><img src="/development/ai-wont-kill-your-job-your-customers-using-ai-will/thumbnail.jpg" style="width:50%; float:right; padding: 10px"></a>
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)